src/pages/fileCentricCart/cartController.js

```diff
@@ -10,14 +11,15 @@ const cartController = () => {
   const ids = cart.fileIds ? cart.fileIds : [];
   const defaultSortDirection = cart.sortDirection === '' ? table.defaultSortDirection || 'asc' : cart.sortDirection;
   const CART_QUERY = defaultSortDirection === 'desc' ? GET_MY_CART_DATA_QUERY_DESC : GET_MY_CART_DATA_QUERY;
+  const defaultSortColumnValue = cart.sortColumn === '' ? table.defaultSortField || '' : cart.sortColumn;
 
   const { loading, error, data } = useQuery(CART_QUERY, {
     variables: { file_ids: ids, order_by: cart.sortColumn === '' ? table.defaultSortField || '' : cart.sortColumn },
   });
 
   if (loading) {
     return (
-      <CartView isLoading data="undefined" />
+      <CircularProgress />
```

Comment:
If you pass a cart view you can show a loading icon within the cart view rather the whole page rerender